otter/models/cass.py

```diff
@@ -206,7 +206,7 @@ def serialize_json_data(data, ver):
 
 _cql_count_for_tenant = (
     'SELECT COUNT(*) FROM {cf} '
-    'WHERE "tenantId" = :tenantId;')
+    'WHERE "tenantId"=:tenantId {deleting};')
```

Comment:
First I wanted to suggest changing {deleting} to something like {extra_clauses}, since it doesn't intrinsically have anything to do with deleting (other than where it's used)...

but then I figured it'd be better to just have two different constant CQL expressions, one for getting deleting groups and one for getting active groups, to keep CQL clause details out of the python code below...

but now I'm wondering why we ever want to get deleting groups in this query? It's not clear to me why `get_counts` should return information about deleting groups.